build-system/tasks/jsify-css.js

```diff
@@ -35,9 +35,16 @@ var cssprefixer = autoprefixer({
   ]
 });
 
+// See http://cssnano.co/optimisations/ for full list.
+// We try and turn off any optimization that is marked unsafe.
 cssnano = cssnano({
+  autoprefixer: false,
```

Comment:
man, this is a misleading name for this options! Had to look up http://cssnano.co/optimisations/autoprefixer/ since I assumed we want this. Apparently has nothing to do with `adding` browser specific prefixes, but it is about removing them..